File: Block Demo/Block Demo/ViewController.m
//
//  ViewController.m
//  Block Demo
//
//  Created by DXC-Technology on 10/31/17.
//  Copyright © 2017 lukasnguyen. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    void (^simpleBlock)(int);
    
    simpleBlock = ^ (int num1) {
        NSString *myString = [NSString stringWithFormat:@"This is %d", num1];
        NSLog(@"%@", myString);
    };
    
    simpleBlock(10);
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: day-2/Label/Label/ViewController.swift
//
//  ViewController.swift
//  Label
//
//  Created by DXC-Technology on 10/25/17.
//  Copyright © 2017 lukasnguyen. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    @IBOutlet weak var lblText: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        lblText.textColor = UIColor.blue
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

